react/spring-page/client/src/utils/useForm.js

```diff
@@ -0,0 +1,32 @@
+import { useState } from 'react';
+import { validateInfo } from './validateInfo';
+
+export const useForm = () => {
```

Comment:
Commendable but stupid (and not flexible). Use proper library like https://react-hook-form.com/